src/get-started/java-binding.md

```diff
@@ -326,6 +377,27 @@ for the given transaction. An observable result is the one that affects the
 blockchain state hash: a modification of a collection that affects the service
 state hash, or an execution exception.
 
+<!-- todo: Shall we put under "Transactions" (as a modifying operation)
+or under "Blockchain Events" (as a handler of 'before commit' event)? -->
+
+#### Block Commit Handler
```

Comment:
This name is confusing given that the "Blockchain Events" section talks about *another* block commit handler. I'd suggest to use a term not including "commit" (since there is no guarantee that the block will be committed when the handler is run). Of course, it's complicated by the fact that the hook itself is called `before_commit` (not for long though, see exonum/exonum#1577). The new name - "after transactions" [handler] - looks more understandable to me.